### PR TITLE
WIP Fix parameter type for PHPDoc

### PR DESCRIPTION
**Pull request by vibbow**
**Description**
Update PHPDoc to correct some parameter type hint.

**Checklist:**
- [ Y ] Securely signed commits
- [ N/A ] Component(s) with PHPdocs
- [ N/A ] Unit testing, with >80% coverage
- [ N/A ] User guide updated
- [ Y ] Conforms to style guide

### CODE REVIEW

File: system/HTTP/Request.php [hunk 1]
```diff
@@ -336,10 +336,10 @@ public function setGlobal(string $method, $value)
 	 *
 	 * http://php.net/manual/en/filter.filters.sanitize.php
 	 *
-	 * @param integer      $method Input filter constant
+	 * @param string       $method Input filter constant
```
Comment (jim-parry) [inline system/HTTP/Request.php]: PHP filters are actually integers

Comment (vibbow) [reply system/HTTP/Request.php]: Oh yes, my bad

File: system/API/ResponseTrait.php [hunk 2]
```diff
@@ -95,7 +95,7 @@
 	 * Provides a single, simple method to return an API response, formatted
 	 * to match the requested format, with proper content-type and status code.
 	 *
-	 * @param null    $data
+	 * @param mixed    $data
```
Comment (jim-parry) [inline system/API/ResponseTrait.php]: wouldn't line 330 be changed to match?

**Comment by jim-parry** [comment]
Good idea. I've left some code comments.
ps - generally a poor idea to make changes/PR directly in your 'develop' branch -> you should always use a feature branch for those, so you can work on more than one at a time without them interfering with each other

File: system/API/ResponseTrait.php (hunk 2 above)
Comment (vibbow) [reply system/API/ResponseTrait.php]: Yes, I'm still working on fix all my IDE warning on system folder.

Comment (jim-parry) [reply system/API/ResponseTrait.php]: Ok. I have added WIP to the PR title ... edit that out when you are done.

**Comment by vibbow** [comment]
Closed it due to I didn't commit on a new pr branch, and get it polluted with my own commit.

**Pull request closed without merging**